src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -281,10 +281,8 @@ def main() -> None:
     parser.add_argument_dst_path(
         "--download_dir", default=Path.cwd(), help="Folder where the data will be downloaded"
     )
-    parser.add_log_arguments(add_log_file=True)
+    parser.add_log_arguments()
```

Comment:
I believe Vasily mentioned if we wanted to have the `add_log_file=True` set by default in our codebase just to have the option if needed/wanted.

Reply:
I removed that one without thinking because it's just a (fast) download so I don't see a use for a log file. I'd just add that option if we actually need to be a detailled introspection at some point.

Though if we do decide to set this as default I'll follow of course

Reply:
I prefer to add it as needed, to be honest. So let's move on.